lib/getPaths.js

```diff
@@ -4,21 +4,39 @@
 */
 "use strict";
 
-module.exports = function getPaths(path) {
+const path = require("path");
+
+const isSubDir = function(parent, dir) {
```

Comment:
This function doesn't fit into this file. But I guess we don't need it anymore when using `startsWith`.